src/seedcase_sprout/core/set_missing_values_to_null.py

```diff
@@ -0,0 +1,40 @@
+import polars as pl
+
+from seedcase_sprout.core.get_nested_attr import get_nested_attr
+from seedcase_sprout.core.properties import (
+    FieldProperties,
+    ResourceProperties,
+)
+
+
+def set_missing_values_to_null(
+    data_frame: pl.DataFrame, resource_properties: ResourceProperties
```

Comment:
```suggestion
    data: pl.DataFrame, resource_properties: ResourceProperties
```

To match naming scheme.